crates/grafbase-sdk/src/test.rs

```diff
@@ -6,9 +6,8 @@
 //! - Building and loading extensions
 
 mod config;
-mod mock;
 mod runner;
 
 pub use config::{TestConfig, TestConfigBuilder};
-pub use mock::{DynamicSchema, DynamicSubgraph};
+pub use grafbase_sdk_mock::{DynamicSchema, DynamicSubgraph};
```

Comment:
Not necessary now, but we might want to wrap `async_graphql::ServerError` or re-export it later

Reply:
I guess you could just do `pub use grafbase_sdk_mock::*`

Reply:
Using a star in a pub use is not cool man...